Libraries/Blob/BlobTypes.js

```diff
@@ -12,14 +12,15 @@
 
 'use strict';
 
-export type BlobProps = {
+export type BlobData = {
   blobId: string,
   offset: number,
   size: number,
   type?: string,
+  lastModified?: number,
```

Comment:
Should we add `createdAt`?

Reply:
@Kureev `File`s only have a `lastModifiedProperty` https://developer.mozilla.org/en/docs/Web/API/File